src/views/Header/HeaderBackButton.js

```diff
@@ -79,15 +80,19 @@ class HeaderBackButton extends React.PureComponent {
 
     const backButtonTitle = renderTruncated ? truncatedTitle : title;
 
-    return (
+    const pressColor = pressColorAndroid
+      ? pressColorAndroid
+      : `rgba(${tintColor}, .32)`;
```

Comment:
this isn't a valid color, using the default values it returns: `rgba(#000000, .32)`

Reply:
Good catch. Do you have any suggestions on how to handle this? Since `tintColor` can be any "kind" of color format (hex, rgb, colorname,..), but the alpha value should be lowered for the ripple. Is a color converter something?

Reply:
react-native uses this function called `normalizeColor` for it: https://github.com/facebook/react-native/blob/d01ab66b47a173a62eef6261e2415f0619fefcbb/Libraries/Color/normalizeColor.js

it is based off of [tinycolor](https://github.com/bgrins/TinyColor). unfortunately `normalizeColor` isn't exposed through the public api, so we might want to pull in `tinycolor` and use it here.

Reply:
And with pull in, you mean install it as a dependency, or just add the file into the utils folder?

Reply:
@ricardobeat I'm a bit out of time, so if you want to adapt, be my guest :-) I've send an invitation

Reply:
@ricardobeat - still interested?